### ISSUE TITLE
v18.4.0 Release

### ISSUE THREAD

**Issue by dennisklein** [FairRootGroup/FairRoot#964]
* [x] Create release candidate branch -> [RC_v18.4.0](https://github.com/FairRootGroup/FairRoot/tree/RC_v18.4.0)
* [x] Write release notes
* [x] Finish/Close milestone [v18.4](https://github.com/FairRootGroup/FairRoot/milestone/2) (merge last minute changes into RC branch)
* [x] Check/Update README
* [x] Test RC against FairSoft

| | `jun19` legacy | `jun19_mt` legacy | kernel | compiler |
| --- | --- | --- | --- | --- |
| `gsi-debian8` | ✅ | ✅ | `Debian 3.16.81-1 x86_64 ` | `gcc 4.9.2` and FairSoft from `/cvmfs/fairroot.gsi.de` |
| `gsi-debian8` | ✅ | ✅ | `Debian 3.16.81-1 x86_64` | `gcc 9.1.0` from `/cvmfs/it.gsi.de` |
| `debian10` | ✅ | ✅ | `3.10.0-1062.18.1.el7.x86_64` | `gcc 8.3.0` |
| `centos7` | ✅ | ✅ | `3.10.0-1062.18.1.el7.x86_64` | `gcc 4.8.5` |
| `opensuse15.0` | ✅ | ✅ | `5.4.18-200.fc31.x86_64` | `gcc 7.4.1` |
| `fedora31` | ✅ | ✅ | `5.4.18-200.fc31.x86_64` | `gcc 9.2.1` |
| `ubuntu18.04` | ✅ | ✅ | `3.10.0-1062.18.1.el7.x86_64` | `gcc 7.5.0` |
| `macOS10.14` | ✅ | ✅ | | `appleLLVM 10.0.1` |
| `macOS10.15` | ✅ | ✅ | | `appleLLVM 10.?` |
* couldn't test `opensuse15.2` because of singularity issue with permissions on docker base image
* `gsi-debian8` (kernel `3.16.81-1 x86_64`) with `gcc 9.1.0`/`gcc 9.3.0` from `/cvmfs/it.gsi.de` and FairSoft option `Optimize (3)` (which is `-O3` for Geant3) leads to [Geant3 segfaults](https://cdash.gsi.de/viewTest.php?onlyfailed&buildid=238325)

* [x] Test experiment repos against RC

| | R3BRoot | CBMRoot | PandaRoot |
| --- | --- | --- | --- | 
| `dev` | ✅ | ✅ | ✅ |
| `stable` | ✅ | ✅ | ✅ |

---
* [x] Once RC is approved, merge into `master` and create a tag `v18.4.0`
* [x] Generate doxygen docs and upload

**Comment by MohammadAlTurany** [comment]
All Done! FairRoot 18.4.0 is released
